### PR TITLE
fix(docs): failure to migrate from closed alpha

### LINKED ISSUE #1174: Unable to install winglang - `Not Found - GET https://npm.pkg.github.com/@winglang%2fsdk`
### Community Note

> Please vote by adding a 👍 reaction to the issue to help us prioritize.
> If you are interested to work on this issue, please leave a comment.

### I tried this:

I tried to follow the instructions to install Wing:

```sh
$ npm i winglang
```

### I expected this:

To work

### Instead, this happened:

```
npm ERR! 404 Not Found - GET https://npm.pkg.github.com/@winglang%2fsdk - npm package "sdk" does not exist under owner "winglang"
```

### Component:

Other

### Wing Version:

0.4.77

### Node.js Version:

_No response_

### Platform(s):

MacOS

### Anything else?

The reason is that my `~/.npmrc` had an entry for @winglang which points to GitHub packages.
The solution is to delete this entry.

### PR DESCRIPTION
**Pull request by eladb**
Update the getting started guide to instruct users on how to upgrade from the closed alpha version (`@winglang/wing`).

Fixes #1174 

*By submitting this pull request, I confirm that my contribution is made under the terms of the 
[Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

**Review by staycoolcall911 — APPROVED** [review]

File: docs/02-getting-started/02-installation.md [hunk 1]
```diff
@@ -26,6 +26,32 @@ In order to deploy to AWS, you will also need:
 
 ## Wing CLI
 
+:::caution If you were in our closed alpha
```
Comment (ShaiBer) [inline docs/02-getting-started/02-installation.md:29]:
```suggestion
<details>
  <summary>caution If you were in our closed alpha</summary>
```

File: docs/02-getting-started/02-installation.md [hunk 2]
```diff
@@ -26,6 +26,32 @@ In order to deploy to AWS, you will also need:
...
+
+Now, edit `~/.npmrc` and remove this line:
+
+```
+@winglang:registry=https://npm.pkg.github.com/
+```
+
+You can also just delete `~/.npmrc` if there are no other registries that you are signed into.
+
+:::
```
Comment (ShaiBer) [inline docs/02-getting-started/02-installation.md:53]: </details>

Comment (ShaiBer) [inline docs/02-getting-started/02-installation.md:53]:
```suggestion
</details>
```

File: docs/02-getting-started/02-installation.md (hunk 1 above)
Comment (eladb) [reply docs/02-getting-started/02-installation.md]: I think it's better to make this very visible for now, and we'll remove it soon enough.

**Review by staycoolcall911 — APPROVED** [review]

**Pull request merged**